### ISSUE TITLE
GoogleLogin button is not responsive

### ISSUE THREAD

**Issue by nachodeh** [MomenSherif/react-oauth#350]
It's currently not possible to make the button take up 100% of the available width. It does initially when rendering but immediately updates to some smaller size. This makes it very hard to make the button fit into any website's style.

**Comment by vasanthmn1** [comment]
Use Custom Google Button  
```
import { useGoogleLogin } from '@react-oauth/google';

const login = useGoogleLogin({
  onSuccess: tokenResponse => console.log(tokenResponse),
});

<MyCustomButton onClick={() => login()}>Sign in with Google 🚀</MyCustomButton>;
```

**Comment by nachodeh** [comment]
> Use Custom Google Button
> 
> ```
> import { useGoogleLogin } from '@react-oauth/google';
> 
> const login = useGoogleLogin({
>   onSuccess: tokenResponse => console.log(tokenResponse),
> });
> 
> <MyCustomButton onClick={() => login()}>Sign in with Google 🚀</MyCustomButton>;
> ```

Thanks, but this doesn't really work because useGoogleLogin provides a tokenResponse and I need a credentialResponse for my API

**Comment by vasanthmn1** [comment]
Currently  accessToken Only return in CustomBtn,not available credential Tokan in the CustomBtn

**Comment by Nicolasbort** [comment]
@nachodeh Have you found a solution so far?

**Comment by nachodeh** [comment]
@Nicolasbort yeah, I stopped using this button, created my own, and implemented token retrieval in my own API. In the end it's much better this way and gives me more flexibility.

**Comment by Nicolasbort** [comment]
@nachodeh that's a good idea, thanks for the reply.

**Comment by nickcan** [comment]
Why does `useGoogleLogin` return an access token instead of a user credential like the GoogleLogin component does?

It just feels odd because now I need to change the backend implementation simply because the button won't fill the width of the container.

**Comment by ciccilleju** [comment]
having the same issue.

**Comment by divya-nsh** [comment]
> Why does `useGoogleLogin` return an access token instead of a user credential like the GoogleLogin component does?
> 
> Thanks btw! I did get it working. It just feels odd because now I need to change the backend implementation simply because the button won't fill the width of the container.

I have Same question

**Comment by NuraiymMamatova** [comment]
It helped me
```
   const login = useGoogleLogin({
           onSuccess: async tokenResponse => {
               const token = tokenResponse.access_token;
             // fetching userinfo can be done on the client or the server
             console.log(token);
             const userInfo = await axios
               .get('https://www.googleapis.com/oauth2/v3/userinfo', 
                 { headers: { Authorization: `Bearer ${tokenResponse.access_token}` },
               })
             const result = userInfo.data;
             console.log(result);
            // contains name, email & googleId(sub)
           },
         });

<MyCustomButton onClick={login}>Sign in with Google 🚀</MyCustomButton>;
```
source: https://stackoverflow.com/questions/75590977/get-token-and-google-id-in-react-oauth-google
